### PR TITLE
feat: add afc streams for writing and reading operations

### PR DESCRIPTION
**Pull request by umutuzgur**

### CODE REVIEW

File: lib/afc/index.js [hunk 1]
```diff
@@ -102,6 +103,16 @@ class AfcService {
     return res.headerPayload.readUInt32LE(0);
   }
 
+  async createWriteStream (path, options) {
```
Comment (mykola-mokhnach) [inline lib/afc/index.js]: I'd rather use another name for the `path` argument, since it may shadow the `path` module

File: lib/afc/streams.js [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+import _ from 'lodash';
+
+class AfcReadableFileStream extends stream.Readable {
+
+  constructor (fileHandle, afcService, options) {
+    super(options);
+
+    this._fileHandle = fileHandle;
+    this._afcService = afcService;
+    this._stopped = false;
```
Comment (mykola-mokhnach) [inline lib/afc/streams.js]: how about this._isDestroyed?

Comment (umutuzgur) [reply lib/afc/streams.js]: Will rename it to `_destroy`

File: lib/afc/index.js (hunk 1 above)
Comment (umutuzgur) [reply lib/afc/index.js]: Makes sense

**Pull request merged**